randovania/games/dread/json_data/Dairon.txt

```diff
@@ -177,7 +194,9 @@ Extra - asset_id: collision_camera_002
   > Dock to EMMI Zone Exit East
       After Dairon - Wide Beam Block by Save Room
   > Above Crumble Blocks
-      Spider Magnet or Walljump (Beginner) or Simple IBJ or Use Spin Boost
+      Trivial
```

Comment:
what even was the thought process here the first time around

Reply:
Looks like this logic was intended to go from down to up, not the other way lmao. Whoops.